TestCases/radiation/p1model/configp1.cfg

```diff
@@ -23,6 +23,7 @@ INC_DENSITY_MODEL= VARIABLE
 INC_ENERGY_EQUATION = YES
 INC_DENSITY_INIT= 0.00597782417156
 INC_TEMPERATURE_INIT= 288.15
+INC_VELOCITY_INIT= (0, 0, 0)
```

Comment:
Why do you have to add this option here? I might have missed that in the code but if the option is not set I understood your changes that the default remains the same, but with out explicitly having containers for the default value? (default here would be 1,0,0)